charts/keycloak/templates/secret-db.yaml

```diff
@@ -8,4 +8,7 @@ metadata:
 type: Opaque
 data:
   {{ include "keycloak.dbPasswordKey" . }}: {{ .Values.keycloak.persistence.dbPassword | b64enc | quote }}
+{{- if .Values.keycloak.persistence.existingSecretUsernameKey }}
```

Comment:
Since this is a breaking change anyways, I'd prefer it if we handled username and password the same and always put both into the secret.

Reply:
Done.